docs/dev/json.rst

```diff
@@ -175,3 +175,62 @@ submitted via the ``additional_info`` element. We're open to see how people
 will make use of these unspecified fields and may decide to formally specify or
 scrub elements in the future.
 
+Artist Statistics JSON
+----------------------
+The JSON documents returned from our API look like the following::
+
+   {
+     "payload": {
+        "user_id": "-- the MusicBrainz ID of the user --",
+        "artist": {
+           "all_time": {
+              "artists": [
+                 "-- Artist stats here --"
```

Comment:
I feel an example of what the artist stats look would be nice here.

Reply:
The example of how the whole response will be has been given at the end. Will that suffice?